lib/extractors/exported.js

```diff
@@ -28,7 +28,8 @@ function walkExported(ast, data, addComment) {
 
   function getComments(data, path) {
     if (!hasJSDocComment(path)) {
-      return [addBlankComment(data, path, path.node)];
+      var added = addBlankComment(data, path, path.node);
+      return added ? [added] : [];
```

Comment:
This avoids running into a NPE later on when `addBlankComment` does not return a value.